src/createTokenBridge.ts

```diff
@@ -34,6 +35,43 @@ import { isCustomFeeTokenAddress } from './utils/isCustomFeeTokenAddress';
 import { WithTokenBridgeCreatorAddressOverride } from './types/createTokenBridgeTypes';
 import { TransactionRequestGasOverrides } from './utils/gasOverrides';
 import { getBlockExplorerUrl } from './utils/getBlockExplorerUrl';
+import { tokenBridgeCreatorABI } from './contracts/TokenBridgeCreator';
+import { getTokenBridgeCreatorAddress } from './utils';
+
+export async function assertTokenBridgeDoesntExist<
```

Comment:
Since we are exporting this function, it would be very useful to add some jsdoc to explain when and why this method should be used.